contracts/token/BiconomyTokenPaymaster.sol

```diff
@@ -663,14 +677,14 @@ contract BiconomyTokenPaymaster is
                 address(feeToken),
                 account,
                 feeReceiver,
-                actualTokenCost + fee
+                ((actualTokenCost * priceMarkup) / PRICE_DENOMINATOR)
```

Comment:
Should calculate this value once in local variable and the use this to avoid doing same calculation three times.

Reply:
done